src/main/resources/templates/charactercreator.html

```diff
@@ -0,0 +1,24 @@
+<html xmlns:th="https://www.thymeleaf.org">
+<head>
+    <link rel="stylesheet" href="./pico.min.css">
+    <title>Getting Started: Character Sheet Creator</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Design your Character!</h1>
+<form action="#" th:action="@{/}" th:object="${character}" method="post">
+    <p>Enter the Name of your Character:</p>
+    <p><input type="text" th:field="*{name}"/></p>
+    <p>Enter the Name of your Profession:</p>
+    <p><input type="text" th:field="*{profession}"/></p>
+    <p>Enter your Strength stat:</p>
```

Comment:
Constraits should be shown to the user, e.g. "Strength (1-20)"

Reply:
Also minimum should be 1 I guess?

Reply:
fixed that.